plugin/src/main/java/com/playmonumenta/epicstructures/managers/RespawnManager.java

```diff
@@ -39,14 +46,23 @@ public void run()
 	private boolean taskScheduled = false;
 	private boolean structuresLoaded = false;
 
-	public final String mZoneLayerName = "Respawning Structures";
-	protected ZoneLayer<RespawningStructure> mZoneLayer = new ZoneLayer<RespawningStructure>(mZoneLayerName);
-	public BaseZoneTree<RespawningStructure> mZoneTree;
+	public final String mZoneLayerNameInside = "Respawning Structures Inside";
+	public final String mZoneLayerNameNearby = "Respawning Structures Nearby";
```

Comment:
These should be static final / all caps constants. Each instance doesn't need a copy (but it doesn't matter much because there's usually only one instance